storage/storage_test.go

```diff
@@ -578,3 +579,48 @@ func TestDBStorage_NewSQLite(t *testing.T) {
 	})
 	helpers.FailOnError(t, err)
 }
+
+func TestDBStorageWriteConsumerError(t *testing.T) {
+	mockStorage := helpers.MustGetMockStorage(t, true)
+	defer helpers.MustCloseStorage(t, mockStorage)
+
+	testTopic := "topic"
+	var testPartition int32 = 2
+	var testOffset int64 = 10
+	testKey := []byte("key")
+	testMessage := []byte("value")
+	testProducedAt := time.Now().Add(-time.Hour).UTC()
+	testError := fmt.Errorf("Consumer error")
+
+	err := mockStorage.WriteConsumerError(&sarama.ConsumerMessage{
+		Topic:     testTopic,
+		Partition: testPartition,
+		Offset:    testOffset,
+		Key:       testKey,
+		Value:     testMessage,
+		Timestamp: testProducedAt,
+	}, testError)
+
+	assert.NoError(t, err)
```

Comment:
it's better to use `helpers.FailOnError(t, err)`

Reply:
@Sergey1011010 Good point. I didn't realize `NoError` didn't immediately fail the test.